src/integrations/windowsIntegrationManager.ts

```diff
@@ -420,13 +455,30 @@ export default class WindowsIntegrationManager implements IntegrationManager {
         },
         executable, 'kubeconfig', '--show');
 
+      console.log(`WSL distro "${ distro.name }: wsl-helper output: "${ stdout }"`);
       if (['true', 'false'].includes(stdout.trim())) {
         return stdout.trim() === 'true';
       } else {
-        return stdout.trim();
+        return `Error: ${ stdout.trim() }`;
       }
     } catch (error) {
-      return (typeof error === 'object' && error?.toString()) || false;
+      if (typeof error === 'object' && error) {
+        const errorString = error.toString();
+
+        console.log(`WSL distro "${ distro.name }: error: ${ errorString }`);
+
+        return `Error: ${ errorString }`;
+      } else if (typeof error === 'string') {
+        console.log(`WSL distro "${ distro.name }: error: ${ error }`);
+
+        return `Error: ${ error }`;
+      } else {
+        const errorString = `unexpected error getting state of distro`;
+
+        console.log(`WSL distro "${ distro.name }: error: ${ errorString }`);
```

Comment:
Shouldn't `error` itself be logged somewhere?

Reply:
Yeah, now that I take another look it doesn't hurt to print it to the logs. I think I was thinking, "sometimes when I print something of unknown type I just get `[object Object]`", but we've handled the `object` case here. Best to keep a response that makes sense for the UI though.

Reply:
Yeah, the UI shouldn't get `error` here, just the logs.